extension/src/store/actions.js

```diff
@@ -77,14 +77,17 @@ export default ({ apollo }) => {
     )
   }
 
-  const deleteAccountWithoutPassword = async ({ commit }, { address }) => {
+  const deleteAccountWithoutPassword = async (store, { address }) => {
     chrome.runtime.sendMessage(
       {
         type: 'DELETE_WALLET_WITHOUT_PASSWORD',
         payload: { address }
       },
-      function (response) {
-        commit('setAccounts', response || [])
```

Comment:
instead you should have send the remaining accounts back from background.js as stated in the issue